src/test/java/com/github/ogomezso/kafka/connect/soap/source/SoapSourceConnectorTest.java

```diff
@@ -27,16 +27,22 @@
 
   private final SoapSourceConnector classToTest = new SoapSourceConnector();
 
+/*  TODO Due to the generous copying of params in taskConfigs - see TODO there - this fails with...
```

Comment:
I'm not sure if it is a error of the implementation but there is a mismatch between the expected value and the actual value that returns the tested class:

 
![image](https://user-images.githubusercontent.com/18402399/127969403-022478ef-c292-4ba3-8344-11e57e910b2b.png)


as you can see on the image, the config now have to values requestMessageFiles and requestMessageFile (task and connector config respectively.

Looking at the code I guess that connector taskconfigs method must return a list of taskConfig objects that contains just a requestFile value since for this iteration one task only takes care of 1 file.

Reply:
In other hand I think that requestMessageFiles is not setting up properly I'm not sure if it is on the test object mother class or on the actual code.

Reply:
It is because the connector config is just copied into task config, then taskConfig() creates the requestMessageFile field and puts filepaths into it - still the 'files' field is carried along. There is a TODO comment in the most recent commit.

Reply:
ok so we  need to added to the test object, the main value of the test is to fail when something change if we just comment it, when we start to develop the todo there will be no way to check if something change.